tcms/testruns/static/testruns/js/get.js

```diff
@@ -211,30 +213,33 @@ function updateExecutionText() {
     window.location.reload(true);
 }
 
-function fileBugFromExecution(run_id, title_container, container, case_id, execution_id) {
-    // todo: this dialog needs to be reimplemented with a Patternfly modal
-    var dialog = new AddIssueDialog({
-        'action': 'Report',
-        'onSubmit': function (e, dialog) {
-            e.stopPropagation();
-            e.preventDefault();
-
-            var tracker_id = dialog.get_data()['bug_system_id'];
-            jsonRPC('Bug.report', [execution_id, tracker_id], function (result) {
-                $('#dialog').hide();
-
-                if (result.rc === 0) {
-                    // unescape b/c Issue #1533
-                    const target_url = result.response.replace(/&amp;/g, '&')
-                    window.open(target_url, '_blank');
-                } else {
-                    window.alert(result.response);
-                }
-            });
-        }
-    });
+function fileBugFromExecution(execution) {
+
+    // remove all previous event handlers
+    $('.one-click-bug-report-form').off('submit')
+
+    // this handler must be here, because if we bind it when the page is loaded.
+    // we have no way of knowing for what execution ID the form is submitted for.
+    $('.one-click-bug-report-form').submit(() => {
+        const trackerId = $('.one-click-bug-report-form #id-issue-tracker').val()
+        jsonRPC('Bug.report', [execution.id, trackerId], result => {
+
+            if (result.rc === 0) {
+                // unescape b/c Issue #1533
+                const targetUrl = result.response.replace(/&amp;/g, '&')
+                window.open(targetUrl, '_blank')
+            } else {
+                window.alert(result.response)
+            }
```

Comment:
Error handling is already done by jsonRPC itself so your callback should be handling a success case unconditionally.

Reply:
Also, is it possible to open a new tab/window without switching back to it ?

Reply:
Will the refresh of the current TE row be a separate PR or together with this one ?

Reply:
> Also, is it possible to open a new tab/window without switching back to it ?

i have to read more about that

> Will the refresh of the current TE row be a separate PR or together with this one ?
 
let's do it another PR

Reply:
> Also, is it possible to open a new tab/window without switching back to it ?

https://stackoverflow.com/a/6897522/6200136 This is supposed to solve it, but does not work on my machine. May be because of an adblocker I user, or because of my browser.

Reply:
OK, let's forget about the windows switching for now. It looks like more trouble than we need. Will return back to this if it is requested.